packages/client/src/components/TransactionTable.js

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { useModalContext } from "contexts";
+import TransactionStatusIcon from "./TransactionStatusIcon";
+
+const maxWidths = {
+  index: 50,
+  status: 140,
+  date: 160,
+  action: 70,
+};
+
+function getUserFriendlyIntent(intent) {
+  const friendlyNameMap = {
+    FlowToken: "FLOW",
+  };
+  const newWords = intent.split(" ").map((word) => {
+    const isNumber = !Number.isNaN(parseFloat(word));
+    // handle numbers but avoid addresses
+    if (isNumber && !word.startsWith("0x")) {
+      return parseFloat(word);
+    }
+    // handle class names
+    if (word.startsWith("A.")) {
+      const chunks = word.split(".");
+      const cadenceClassName = chunks[chunks.length - 2];
+      return friendlyNameMap[cadenceClassName] ?? cadenceClassName;
+    }
+
+    return word;
+  });
+
+  return newWords.join(" ");
+}
+
+const TransactionDetails = ({ transaction, onClose }) => {
+  const date = new Date(transaction.eventDate);
+  return (
+    <>
+      <div className="p-5 has-text-black">
+        <h2 className="is-size-4">Transaction details</h2>
+        <p className="has-text-grey">
+          Executed on {date.toLocaleDateString("en-us")} at {date.toLocaleTimeString("en-us")}
+        </p>
+      </div>
+      <div className="border-light-top p-5">
+        <div className="flex-1 is-flex is-flex-direction-column">
+          <div className="is-flex is-align-items-center mt-4">
+            <button className="button flex-1 p-4" onClick={onClose}>
+              Close
+            </button>
+          </div>
+        </div>
+      </div>
+    </>
+  );
+};
+
+const Column = ({ className = "", children, style = {} }) => (
+  <div className={`column is-flex is-align-items-center ${className}`} style={style}>
+    {children}
+  </div>
+);
+
+const Row = ({ transaction, displayIndex, isLastInList, onView }) => {
+  const classes = [
+    "py-4 is-flex is-align-items-center is-justify-content-space-between",
+    isLastInList ? "" : "border-light-bottom",
```

Comment:
We don't need the prop isLastInList, we can use class .table-border for parent element